configs/Asus/SABERTOOTH-P67.CONF

```diff
@@ -0,0 +1,93 @@
+# lm-sensors configuration for the ASUS Sabertooth P67 motherboard
+#
+# Manufacturer: ASUSTeK COMPUTER INC.
+# Product Name: Sabertooth P67
+# Version: Rev 1.2
+# Super I/O: NCT6776
+# UEFI versions tested: 2302
+#
+# Recommended drivers: coretemp, nct6776
+
+# Piet van Duivenbode, 2021
+#
+
+chip "coretemp-*"
+
+  label temp1 "CPU Package DTS"
+  label temp2 "CPU Core 1 DTS"
+  label temp3 "CPU Core 2 DTS"
+  label temp4 "CPU Core 3 DTS"
+  label temp5 "CPU Core 4 DTS"
+  label temp6 "CPU Core 5 DTS"
+  label temp7 "CPU Core 6 DTS"
+  label temp8 "CPU Core 7 DTS"
+  label temp9 "CPU Core 8 DTS"
+
+  
+  chip "nct6776-*"
+
+    label in0 "CPU Core Voltage"
+    
+    label in1 "ATX +12 V Voltage"
+    compute in1 @ / (1 / (11+1)), @ * (1/(11+1))
+    set in1_min  12 * 0.95 # ATX 2.4
+    set in1_max  12 * 1.05 # ATX 2.4
+    
+    label in2 "AVCC Analog Voltage"
+    set in2_min  3.3 * 0.95
+    set in2_max  3.3 * 1.05
+            
+    label in3 "ATX +3.3 V Voltage"
+    set in3_min  3.3 * 0.95
+    set in3_max  3.3 * 1.05
+    
+    label in4 "ATX +5 V Voltage"
+    compute in4 @ / (3 / (12+3)), @ * (3/(12+3))
+    set in4_min  5 * 0.95
+    set in4_max  5 * 1.05
+    
+    #Schematic calls this voltage O_VCORE_OVER dont know what it is. ignored
+    #label in5 "O_VCORE_OVER"
+    #compute in5 @ / (8.2 / (8.2 + 8.2)), @ * (8.2 / (8.2 + 8.2))
+    ignore in5
+    
+    ignore in6 
+    
+    label in7 "3VSB Standby Voltage"
+    set in7_min  3.3 * 0.90
+    set in7_max  3.3 * 1.10
+    
+    
+    label in8 "RTC Battery Voltage"
+    set in8_max 3.4  # from random CR2032 datasheet
+    set in8_min 2.0  # from Intel 200-series chipset spec
+          
+     
+label temp1 "Board Temperature"
+
+  # CPU package temperature from the PECI
+  label temp7 "CPU Temperature"
+
+  ignore temp2   # not actually connected to the CPU/unknown
+  ignore temp3   # not functional/arbitrary value
+  ignore temp4   # not functional/arbitrary value
+  ignore temp5   # not functional/arbitrary value
+  ignore temp6   # not functional/arbitrary value
+  ignore temp8   # not functional/arbitrary value
+
+  ignore temp9   # not functional/arbitrary value
+  ignore temp10  # not functional/always zero
+  
+  # note: PCH Core temperature is not monitored by this chip
+
+
+
+label fan1 "Chassis fan 1"
+label fan2 "CPU fan"
+label fan3 "Chassis fan 2"
+label fan4 "Power Fan"
+label fan5 "Assistant fan"
+
```

Comment:
some extra newlines below (also randomly above btw) that probably shouldnt' be here.

mind the final EOL :)

Also I think after fixing things, I bet the maintainers prefer a single commit in this case, rather then a few 'fixup' commits ...

just my 2 cents :)

Reply:
thanks for your comments :)